mwctl/Makefile

```diff
@@ -0,0 +1,39 @@
+
+export GOPATH=$(CURDIR)/.build
```

Comment:
Why adding the mwctl stuff here? Haven't we got [wikimedia/mwctl](https://github.com/wikimedia/mwctl) for that?